media-tv/mythtv/mythtv-32.0-r2.ebuild

```diff
@@ -5,7 +5,7 @@ EAPI=8
 
 PYTHON_COMPAT=( python3_{8,9} )
 
-inherit flag-o-matic python-any-r1 qmake-utils readme.gentoo-r1 systemd user-info
+inherit flag-o-matic java-pkg-opt-2 java-ant-2 python-any-r1 qmake-utils readme.gentoo-r1 systemd user-info
```

Comment:
I'm not familiar with java stuff in gentoo at all, but why is *java-ant-2* needed here? Thinking about it, *java-pkg-opt-2* should be enough.

Reply:
The configure file in mythtv uses ant. 

From the configure file:
`die "ERROR: BD-J requires ANT, but ant was not found. Please install it."`